sdk/core/azure-core/azure/core/pipeline/policies/_utils.py

```diff
@@ -80,6 +79,7 @@ def get_retry_after(response):
     :return: Value of Retry-After in seconds.
     :rtype: float or None
     """
+    from requests.structures import CaseInsensitiveDict
```

Comment:
I would move the following code into `azure.core.pipeline._tools` and use it here instead:
https://github.com/Azure/azure-sdk-for-python/blob/a30c65fdb462de992e044cb5c26eb3bde3f19c8c/sdk/core/azure-core/azure/core/pipeline/transport/_base.py#L88-L111